tests/openthread_ncp/main.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2017 Baptiste CLENET
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @file
+ * @brief       OpenThread test application
+ *
+ * @author      Baptiste Clenet <bapclenet@gmail.com>
+ */
+
+#include <stdio.h>
+
+#include "ot.h"
+
+int main(void)
+{
+    return 0;
```

Comment:
This looks weird, I though one could use the shell or have some welcome message here ?